src/adapters/scene-stream-access-manager.ts

```diff
@@ -95,13 +95,31 @@ export async function createSceneStreamAccessManagerComponent({
     return result.rowCount > 0 && result.rows[0].streaming
   }
 
+  async function getActiveStreamings(): Promise<SceneStreamAccess[]> {
+    const result = await database.query<SceneStreamAccess>(
+      SQL`SELECT * FROM scene_stream_access WHERE active = true AND streaming = true`
```

Comment:
What do you think about retrieving only the columns that you need from it?
In this case, it would be `created_at` and `ingress_id`, right?